ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -147,6 +147,13 @@
   public final double routerLatencyToleranceQuantile;
 
   /**
+   * The version to use for new BlobIds.
+   */
+  @Config("router.blobid.current.version")
+  @Default("2")
```

Comment:
Consider just making the default value a static final field in this config class. That would get rid of the need for CommonTestUtils.